SonarQube.Client/Models/ServerSentEvents/ClientContract/ITaintServerEvent.cs

```diff
@@ -18,23 +18,24 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using SonarQube.Client.Models.ServerSentEvents.ServerContract;
-
 namespace SonarQube.Client.Models.ServerSentEvents.ClientContract
 {
     public interface ITaintServerEvent : IServerEvent
     {
+        string ProjectKey { get; }
```

Comment:
Do these fields exist on ITaintVulnerabilityClosedServerEvent ? 
I think it would be better to put these fields on ITaintVulnerabilityRaisedServerEvent for readability.

Reply:
Yes, they were. You can see it in the diff: SonarQube.Client/Models/ServerSentEvents/ServerContract/ITaintVulnerabilityClosedSqServerEvent.cs